libs/dal/src/repositories/subscriber/subscriber.schema.ts

```diff
@@ -27,8 +27,8 @@ const subscriberSchema = new Schema(
   schemaOptions
 );
 
-subscriberSchema.index({ _environmentId: 1, userId: 1 });
```

Comment:
We have no userId in this schema... Unless I am missing anything this complex index is not needed.

Reply:
Good catch